src/client.c

```diff
@@ -65,26 +81,43 @@ urlinfo_t *parse_url(char *url)
 int send_request(int fd, char *hostname, char *port, char *path)
 {
   const int max_request_size = 16384;
+  char send_buff[1025];
   char request[max_request_size];
   int rv;
+  // printf("send hostname")
+  sprintf(request,  "GET /%s HTTP/1.1\nHost: %s:%s\r\nConnection: close\r\n\r\n", path, hostname, port);
```

Comment:
We can use the `rv` variable declared up top and assign the return value from `sprintf()` to it. Then `return rv` at the bottom.